examples/tutorials/Introduction_to_PROTACs.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>Formal citations instead of links</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4000/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>